app/lib/import_export.rb

```diff
@@ -378,14 +716,130 @@ def jurisdiction_paths(patients)
     patients_jurisdiction_paths
   end
 
-  # Hash containing mappings between jurisdiction id and name for each patient
-  def jurisdiction_names(patients)
-    jurisdiction_names = Hash[Jurisdiction.find(patients.pluck(:jurisdiction_id).uniq).pluck(:id, :name).map { |id, name| [id, name] }]
-    patients_jurisdiction_names = {}
-    patients.each do |patient|
-      patients_jurisdiction_names[patient.id] = jurisdiction_names[patient.jurisdiction_id]
+  # Extracts assessment data values given relevant fields
+  def extract_assessments_details(patients_identifiers, assessments, fields)
+    if fields.include?(:symptoms)
+      conditions = ReportedCondition.where(assessment_id: assessments.pluck(:id))
```

Comment:
Add some nil checks here, to make sure if we for some reason failed to find ReportedConditions or Symptoms for a given assessment, this won't crash and burn.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/18f2cd1c1295b23fca3eb469bf542092217c6acb#diff-c34521fcd0fdc2034d456886aae8fd35299fe908ab88d8e49c60d0b4d39ab26bR703